src/Swarm/Language/Syntax.hs

```diff
@@ -104,6 +107,9 @@ instance ToJSONKey AbsoluteDir where
 instance FromJSONKey AbsoluteDir where
   fromJSONKey = genericFromJSONKey defaultJSONKeyOptions
 
+-- | A relative direction is one which is defined with respect to the
```

Comment:
I like this definition, it makes sense to have `DDown` relative this way. 👍

Reply:
Yes, I was confused about `DDown` so I added these comments after I figured it out. :smile: